### ISSUE TITLE
Update tvdb and tmdb plugins

### ISSUE THREAD

**Issue by ghost** [koying/SPMC#920]
Looks like both the tvdb and tmdb plugins have been updated to not be update-able for quite some time.

https://github.com/koying/SPMC/commit/ec1a0c49a96fa02665a1ae9da48a9fba3ec72157
https://github.com/koying/SPMC/commit/6b2eb09b8bc23fe192d878f14093bdb5d080e226

This is now causing some issues due to some api changes on the site which is causing the scraping to fail on some items.

These plugins have already been updated for Jarvis/Krypton to fix the issue, but I can't update them in spmc via the addon updater or manual zip from the kodi repo.

**Comment by BigOkie** [comment]
Yeah, it would appear that manual updating reports back that the latest version (in the check for updates) returns back with what it thinks is installed, but the display on what is actually installed winds up always being 3.9.0.  I can work around it for now by using NFO files with the IMDB URLs of movies but would rather not.

**Comment by 78suited** [comment]
I had the same problem and fixed it using the Add-on: Universal Movie Scraper

**Comment by BigOkie** [comment]
That still doesn't work...both of those add ons are being prevented from being updated for whatever reason.

**Comment by 78suited** [comment]
It works for me, just use IMDB as the source in the Universal Movie Scraper. I've noticed it recently and formatted my Nvidia Shield because I thought it was Shield related. Only a couple of movies were "scraped" and all the other ones weren't recognized. After installing and configuring Universal Movie Scraper and selecting it for my video files, it worked like a charm for me.

**Comment by BigOkie** [comment]
OK, maybe I'm wrong..I have my content source set to use the TMDB addon..i'm changing it to the universal to see if that fixes it.

**Comment by 78suited** [comment]
Just let us know if it worked for you also, I think it should....

**Comment by BigOkie** [comment]
It's scanning my library now...sofar it seems to be working.

EDIT:  I had it rescan my library (about 45 movies sofar) it took about 10 minutes.  Thanks for the tip.

**Comment by 78suited** [comment]
There you go, it took me almost a day to figure out what happened and get it "fixed".

**Comment by shavanni** [comment]
the Universal Movie Scraper doesn't seem to be selectable for TV shows. Its only available for Movie content so we need TVDB scraper working again please

**Comment by ghost** [comment]
So for the movies I kept using tmdb but enabled the create nfo option in couchpotato then moved my library, and re imported it and couchpotato created the nfo's and all was well. For TV there is the metadata option in sonarr if you are using sonarr which might help. So far I haven't been effected by any tv issues yet so I haven't tested it out.

**Comment by thomfam** [comment]
I also have a copy of Kodi on my box and used the export Library option on it.  It then leaves a small info file with each movie file.  Reload SPMC and all the library files now appear.

**Comment by lelin88** [comment]
Seems TVDB server is down
http://www.isitdownrightnow.com/thetvdb.com.html

**Comment by tankalf** [comment]
I have a Nvidia Shield.
From some days, my films library and tv-series library on my Nas doesn't update anymore.
I thought it was a Shield problem, instead I know now it's a API problem.

What can be done to solve this issue? Specially for TV-Series?
Thanks a lot

**Comment by 78suited** [comment]
@tankalf look for the solution at my posts earlier in this thread (I use a Shield too btw).

**Comment by tankalf** [comment]
@78suited thanks for your reply.
I read it, but I think to understand it works only for movies library, isn't it?
And for TV-Series library?

Thanks a lot

**Comment by lelin88** [comment]
TVDB is up and running, no more problems
Worked for me today

**Comment by tankalf** [comment]
@78suited Thanks! I tried the Universal Scraper and it works great!
For TVDB it's right, it works now!

Thanks a lot

**Comment by shavanni** [comment]
The TVDB scraper is still down though

**Comment by docgonzo2015** [comment]
hi! I've got a connection failure to tmdb. is there a api problem?

**Comment by tankalf** [comment]
I confirm it's impossible to scrap images, posters, and any other image from TMDB... Any solution?

**Comment by ghost** [comment]
I used these addon updates on the current stable 16.7.1

http://www.minixforum.com/threads/scan-new-content-no-longer-gets-movie-tv-series-info.17454/page-2
http://www.minixforum.com/threads/scan-new-content-no-longer-gets-movie-tv-series-info.17454/page-3

And was able to get things going again.
